libraries/ms-common/test/test_gpio_extern.c

```diff
@@ -0,0 +1,181 @@
+#include <netinet/in.h>
+#include <stdbool.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
+#include <errno.h>
+
+#include "delay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "status.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#define PORT 4000
```

Comment:
Please make sure your naming meets our style guide (this holds for the rest of your code)